serverlessworkflow/modules/ROOT/pages/getting-started/cncf-serverless-workflow-specification-support.adoc

```diff
@@ -1 +1,368 @@
-= CNCF Serverless Workflow specification support
\ No newline at end of file
+CNCF Serverless Workflow Specification Support
+==============================================
+v1.0, 06-08-2022
+:compat-mode!:
+// Metadata:
+:description: CNCF Serverless Workflow Specification Support
+:keywords: kogito, workflow, serverless, cncf, specification
+// links
+:quarkus_config_guide_url: https://quarkus.io/guides/config-reference
+
+
+In this guide, you can learn the details about the Kogito implementation of the {spec_website_url}[CNCF Serverless Workflow] specification.
+
+== Current Status
+
+Currently, Kogito implements version {spec_doc_url}[{spec_version}] of the specification. The table below classifies the current implementation status for each specification feature. {spec_doc_url}#Retry-Definition[Retries] are the only feature that Kogito does not support yet.
+
+.Current implementation status of the specification features
+[cols="1,1,1"]
+|===
+| Feature | Status | Spec Reference
+
+| <<states>>
+| emoji:full_moon[]  
+| {spec_doc_url}#workflow-states[Workflow States]
+
+| <<functions>>
+| emoji:last_quarter_moon[] 
+| {spec_doc_url}#Function-Definition[Function Definition]
+
+| <<events>>
+| emoji:last_quarter_moon[] 
+| {spec_doc_url}#Event-Definition[Event Definition]
+
+| <<workflow_data>>
+| emoji:full_moon[]  
+| {spec_doc_url}#Workflow-Data[Workflow Data]
+
+| <<expressions>>
+| emoji:full_moon[] 
+| {spec_doc_url}#Workflow-Expressions[Workflow Expressions]
+
+| <<error_handling>>
+| emoji:full_moon[] 
+| {spec_doc_url}#Workflow-Error-Handling[Workflow Error Handling]
+
+| <<retries>>
+| emoji:construction[]
+| {spec_doc_url}#Retry-Definition[Retry Definition]
+
+| <<timeouts>>
+| emoji:construction[]
+| {spec_doc_url}#workflow-timeouts[Workflow Timeouts]
+
+| <<compensation>>
+| emoji:full_moon[]
+| {spec_doc_url}#Workflow-Compensation[Workflow Compensation]
+
+| <<constants>>
+| emoji:full_moon[]
+| {spec_doc_url}#workflow-constants[Workflow Constants]
+
+| <<secrets>>
+| emoji:full_moon[]
+| {spec_doc_url}#workflow-secrets[Workflow Secrets]
+|===
+
+The following sections describe the current status of the supported features.
+
+[#states]
+=== Workflow Model - States
+
+Kogito supports every state described in the specification {spec_version} version.
+
+.Workflow States implementation status detail
+[cols="1,1,1"]
+|===
+| State | Status | Spec Reference
+
+| Event
+| emoji:full_moon[] 
+| {spec_doc_url}#Event-State[Event State]
+
+| Operation
+| emoji:full_moon[] 
+| {spec_doc_url}#Operation-State[Operation State]
+
+| Switch
+| emoji:full_moon[] 
+| {spec_doc_url}#Switch-State[Switch State]
+
+| Delay
```

Comment:
Is it Delay state or Sleep state? I often see them interchanged. What should we be using? The link doesn't work because it should be "https://github.com/serverlessworkflow/specification/blob/0.8.x/specification.md#sleep-state"

Reply:
Is it really fully implemented? Someone commented in the constructs document that it also requires JobService so not sure if we should consider it done.

Reply:
Oh yeah, it was a mistake. I'll fix it.